l1-contracts/.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "lib/forge-std"]
```

Comment:
we are using nix for handling external dependency

Reply:
removed the dependency on git modules.